src/download_helper.js

```diff
@@ -0,0 +1,25 @@
+function generateDownloadList(){
+    filelist = "";
+    $('a[href^="https://p4.bcbits.com/download/"]').each(function(index, item){
+        url = $(item).attr('href');
+        filelist += "curl -OJ " + url + " \\ &\n";
+    })
+    return filelist
+};
+
+function download(filename, text) {
+  var element = document.createElement('a');
+
+  element.setAttribute('href', 'data:text/plain;charset=utf-8,' + encodeURIComponent(text));
```

Comment:
Consistency nit: `generateDownloadList()` uses jQuery for DOM traversal, while `download()` uses raw JS. The `download()` method could use jQuery to create the anchor element a bit more cleanly, or the generate method could use native JS to obtain `filelist`. Inconsequential either way, so long as jQuery is globally available.